source/common/event/dispatcher_impl.cc

```diff
@@ -1,11 +1,10 @@
-#include "dispatcher_impl.h"
-#include "file_event_impl.h"
-#include "signal_impl.h"
-#include "timer_impl.h"
-
 #include "envoy/network/listener.h"
 #include "envoy/network/listen_socket.h"
 
+#include "common/event/dispatcher_impl.h"
```

Comment:
Keep related header on the top? (throughout)
https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes